src/Serilog.Sinks.File.Archive/ArchiveHooks.cs

```diff
@@ -70,12 +85,62 @@ public override void OnFileDeleting(string path)
                         sourceStream.CopyTo(compressStream);
                     }
                 }
+                //only apply archive file limit if we are archiving to a non tokenised path (constant path)
```

Comment:
I'm not particularly keen on having some features only working with tokenised paths, but I don't see any way for this to work with tokenised paths 🤷

Reply:
Agree, we need more information exposed from the file sink to be useful in "advanced" usages, but this works for my scenario where I don't need/use a separate directory.